.eslintrc.json

```diff
@@ -6,15 +6,22 @@
     "airbnb-base/legacy"
   ],
   "globals": {
+    "afterEach": true,
+    "beforeEach": true,
+    "afterAll": true,
+    "beforeAll": true,
     "describe": true,
+    "expect": true,
     "it": true,
     "fit": true,
-    "expect": true,
-    "xit": true,
+    "test": true,
     "xdescribe": true,
-    "fdescribe": true
-  }, 
+    "fdescribe": true,
+    "xit": true,
+    "xtest": true
+  },
   "rules": {
+    "linebreak-style": "off",
```

Comment:
My git is set to change the `eol` to `CRLF` when `pull` and `LF` when `push`as I'm on `Windows`. The rule is colliding with that. There isn't going to be an issue if we disable the rule.